logstash-offline/roles/logstash-ansible/tasks/install-logstash.yaml

```diff
@@ -22,20 +22,18 @@
   become: yes
   command: mv /etc/temp/logstash-7.17.1 /etc/logstash
 
-- name: Create dirs
+- name: Create conf.d
   become: yes
   file:
-    path: /etc/logstash/"{{ item }}"
+    path: /etc/logstash/conf.d
     state: directory
-  with_items:
-    - conf.d
-    - logs
 
 - name: Add conf files
   become: yes
   copy:
     src: "{{ item }}"
     dest: /etc/logstash/conf.d/
+    mode: 0700
```

Comment:
This permission should 0755 since they do not need to be hard restricted to user access only.